cmd/gateway/main.go

```diff
@@ -169,7 +162,14 @@ func runGateway() error {
 }
 
 func main() {
-	if err := runGateway(); err != nil {
+	var opts commandlineOpts
```

Comment:
I moved this around a bit, because the commandline help information was being printed out twice.